testinfra/modules/file.py

```diff
@@ -169,6 +169,18 @@ def size(self):
         """Return size of file in bytes"""
         raise NotImplementedError
 
+    @property
+    def list_dir(self):
+        """Return list of items under the directory
+
+        >>> host.file("/tmp").list_dir
+        ['foo_file', 'bar_dir']
+        """
+        out = self.run_test("ls -- %s", self.path)
+        if out.rc != 0:
+            raise RuntimeError("Unexpected output %s" % (out,))
+        return out.stdout.splitlines()
```

Comment:
I think this will not work for files containing spaces (or even \r or \n) in their name. Maybe `ls -1` (and additional option) or `find` command has a way to parse them easily, maybe using \0 (null) separator ?

Reply:
very good point. I used `ls -q` to handle new lines (7f80bb13c24a7d1dc092a4b690f4c528e04ffd85 commit message tells more)